db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/limits.rb

```diff
@@ -0,0 +1,90 @@
+#
+# Description: Get quota values.
+#
+
+QUOTA_ATTRIBUTES = %w(storage vms cpu memory)
+
+def quota_values(model_attribute, tag_name)
+  $evm.log(:info, "Getting Quota Values for Model: #{model_attribute} Tag Name: #{tag_name}")
+  object_value = quota_model_value(model_attribute) unless $evm.parent.nil?
+  tag = @source.tags(tag_name).first
+  tag_value = tag_value(tag_name, tag) || 0
+  final_quota_value(tag_value, object_value)
+end
+
+def tenant_quota_values
+  tenant_values = {}
+  @source.tenant_quotas.each do |q|
+    value = q.value.to_i
+    name = q.name.chomp('_allocated')
+    name = 'memory' if name == 'mem'
+    tenant_values[name.to_sym] = (name == 'memory' || name == 'storage') ? bytes_to_megabytes(value) : value
```

Comment:
@tinaafitz 
From the UI we expect user input in Giga bytes, they are converted to mega bytes and stored in the DB. So I think the values for the tenant quota is already in MB and we don't have to convert. The conversion might be needed when we get the used values in bytes and convert it to MB before we do the comparison.